sdk/src/main/java/com/horizen/account/transaction/EthereumTransaction.java

```diff
@@ -221,7 +221,7 @@ else if (this.isSigned()) {
             var sigData = signedTx.getSignatureData();
             if (sigData.getS()[0] == 0 && sigData.getR()[0] == 0)
                 return convertToLong(sigData.getV());
-            else return ((SignedRawTransaction) this.transaction).getChainId(); // TODO check necessity
+            else return ((SignedRawTransaction) this.transaction).getChainId(); // useful for EIP155
```

Comment:
do we really need if/else here? Can't we use only `((SignedRawTransaction) this.transaction).getChainId()` ?

Reply:
I added some comment